### PR TITLE
Feature/seed database

### PR DESCRIPTION
**Pull request by hoihmy**
I cannot pull request from feature/seed-database to develop.

### CODE REVIEW

**Comment by vuongnd** [comment]
Please check the conflict before merging!

**Comment by vuongnd** [comment]
Firstly:
   Pull latest code in develop and your current branch.
Then:
   merge develop to your local branch. and resolve conflict.
After that:
   Push the new change in your local branch
   The Pull request are open.
   We only accept it after all issues was be fixed.

**Comment by hoihmy** [comment]
Yes, I will pull lastest code form develop and pull request again. tks

**Comment by hoihmy** [comment]
Pls, review again and merge it. tks

File: Rose/database/migrations/2014_10_12_000000_create_users_table.php [hunk 1]
```diff
@@ -23,6 +23,14 @@ public function up()
             $table->rememberToken();
             $table->timestamps();
         });
+
+        DB::table('users')->insert([
```
Comment (nhanhvasnet) [inline Rose/database/migrations/2014_10_12_000000_create_users_table.php]: Don't seed values for the database in migration file. 
Please move it to seed file.

**Pull request merged**